docs/docs/50-running/06-linking-acorns.md

```diff
@@ -2,16 +2,39 @@
 title: Linking Acorn Apps
 ---
 
-Acorns can be linked with other running Acorns at runtime to provide supporting services. For instance if you have an Acorn running Postgresql, it can be used to provide the `db` service to another app.
+Acorn apps can link to containers from other Acorn apps at runtime to provide supporting services. For instance, if you have an Acorn running PostgreSQL, it can be used to provide the `db` service to another app.
```

Comment:
I heard talk of getting rid of the reference to "apps." Everything is an Acorn.

Reply:
Imo for now it should be fine to keep using this language in the docs - that's what the API type is still called, and we'll have to overhaul the docs if we do choose to abandon the term in the future.